app/helpers/course/assessment/question/scribing_helper.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+module Course::Assessment::Question::ScribingHelper
```

Comment:
Don't think this helper is necessary. Upon validation failure, the React code should handle the failure and display messages there.